src/chocolatey/infrastructure.app/configuration/ChocolateyConfiguration.cs

```diff
@@ -497,6 +506,17 @@ public sealed class ApiKeyCommandConfiguration
     }
 
     [Serializable]
+    public sealed class PackCommandConfiguration
+    {
+        public PackCommandConfiguration()
+        {
+            Properties = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);

```

Comment:
Ah fair - we like to tell folks the things we like as well as what we see that we that has questions, comments, or needs changes. Some things come down to differences in preferences and don't require any changes, just comments.